### PR TITLE
Admin panel po

### PR DESCRIPTION
**Pull request by martadiachok**
Add Admin panel

### CODE REVIEW

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+import org.openqa.selenium.WebElement;
+import java.util.List;
+import com.easyrest.components.Calendar;
+
+public class AdminPanel extends BasePage {
+
+    public AdminPanel(WebDriver driver) {
+        super(driver);
+    }
+
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/pages/AdminPanel.java]: Part of Class Declaration

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        clickBannedButton();
+        return this;
+    }
+
+    public AdminPanel banUser(){
+        clickActiveUsers();
+        String userName = getFirstNameFromClient();
+        clickToBanFirstClient();
+        checkIfClientMovedToBannedList(userName);
+
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/pages/AdminPanel.java]: Code Formatting

**Review by vadymtsehelny — APPROVED** [review]

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+        String userName = getFirstNameFromClient();
+        clickToMakeFirstClientActive();
+        checkIfClientMovedToActiveList(userName);
+
+        return this;
+    }
+
+    /**
+     * Methods to deal with owners
+     */
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/pages/AdminPanel.java:125]: I like it, informative comments)

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 4]
```diff
@@ -34,7 +32,7 @@ public AdminPanel(WebDriver driver) {
     private By birthDateField = By.xpath("//input[@name='birthDate']");
     private By passwordField = By.xpath("//input[@name='password']");
     private By confirmPasswordField = By.xpath("//input[@name='repeated_password']");
-    private By createAccountButtonField = By.xpath("//span[text()='Create account']");
+    private By createAccountButton = By.xpath("//span[text()='Create account']");
     private By cancelButton = By.xpath("//span[text()='Cancel']");
```
Comment (ViktoriaMilkevich) [inline src/main/java/com/easyrest/pages/AdminPanel.java]: Locators should private final

**Review by Zeicks — APPROVED** [review]

**Review by ViktoriaMilkevich — APPROVED** [review]

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+    public String getFirstNameFromClient() {
+        List<WebElement> clientList = driver.findElements(usersName);
+        WebElement firstNameFromList = clientList.get(0);
+        return firstNameFromList.getText();
+    }
+
+    public AdminPanel clickToMakeFirstClientActive(){
+        clickBannedButton();
+        List<WebElement> userList = driver.findElements(actionButton);
```
Comment (dmatsev) [inline src/main/java/com/easyrest/pages/AdminPanel.java:47]: this is a duplicated logic, please avoid this

File: src/main/java/com/easyrest/pages/AdminPanel.java [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        if(userList.size() != 0) {
+            userList.get(0).click();
+        } else {
+            System.out.println("There is no one banned users.");
```
Comment (dmatsev) [inline src/main/java/com/easyrest/pages/AdminPanel.java]: throw exception instead

**Review by dmatsev — APPROVED** [review]

**Pull request merged**